README.md

```diff
@@ -34,18 +28,53 @@ npm install react-image --save
 
 ## Dependencies
 
-`react-image` has no external dependencies, aside for the usual `react` and `react-dom`.
+`react-image` has no external dependencies, aside for a version of `react` and `react-dom` which support hooks.
 
 ## Documentation
 
+You can use the standalone component, documented below, or the `useImage` hook.
+
+### useImage():
+
+The `useImage` hook allows for incorperating `react-image`'s logic in any component. When using the hook, the component can be wrapped in `<Suspense>` to keep it from rendering until the image is ready. Specify the `fallback` prop to show a spinner or any other component to the user while the browser is loading. The hook the throw an error if it failes to find any images. You can wrap your componenet with an [Error Boundry](https://reactjs.org/docs/code-splitting.html#error-boundaries) to catch this scenario and do/show something.
+
+Example usage:
+
+```js
+import useImage from 'react-image/useImage'
```

Comment:
> i meant if you are using `react-image/useImage`, the file will need to be produced somewhere to be shipped to npm
> currently im not seeing where that is happening

Should build and run `npm pack` to see what will be sent to npm.
if the package is not whats intended then rollup needs to be configured to produce your use case scenario.

the simpler option as mentioned before is the interface seen below. You would of course need to list it as a breaking change

`import {Image, useImage} from 'react-image'`

Reply:
It seems to me the file is being produced. Am I missing something?


```
$ npm pack
npm notice
npm notice 📦  react-image@2.4.0
npm notice === Tarball Contents ===
npm notice 1.1kB LICENSE
npm notice 5.3kB cjs/index.js
npm notice 2.2kB umd/index.js
npm notice 5.5kB cjs/react-image.js
npm notice 3.7kB cjs/useImage-b693a49d.js
npm notice 170B  cjs/useImage.js   <----------------------------- here
npm notice 3.3kB package.json
npm notice 2.7kB CHANGELOG.md
npm notice 735B  CONTRIBUTING.md
npm notice 9.9kB README.md
npm notice 590B  UPGRADE.md
npm notice 184B  imagePromiseFactory.d.ts
npm notice 53B   index.d.ts
npm notice 790B  react-image.d.ts
npm notice 307B  useImage.d.ts
npm notice === Tarball Details ===
npm notice name:          react-image
npm notice version:       2.4.0
npm notice filename:      react-image-2.4.0.tgz
npm notice package size:  11.0 kB
npm notice unpacked size: 36.5 kB
npm notice shasum:        bf577c16f972f598414908107b59039e1584282d
npm notice integrity:     sha512-FHlbhB0EluEJ2[...]+mKWOrDfyIzlA==
npm notice total files:   15
npm notice
```

> the simpler option

Totally agreed and would like to do that, just trying to prevent a seriously breaking change. Guess I need to understand more why rollbar complains about mixing default and regular exports...

Reply:
ah good! then all is good from my perspective. 
works from my use case 👍

Reply:
ready to ship?